dns-discovery/src/main/java/org/elasticsearch/discovery/srv/SrvDiscoveryModule.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2015 Grant Rodgers
+ *
+ *     Permission is hereby granted, free of charge, to any person obtaining
+ *     a copy of this software and associated documentation files (the "Software"),
+ *     to deal in the Software without restriction, including without limitation
+ *     the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ *     and/or sell copies of the Software, and to permit persons to whom the Software
+ *     is furnished to do so, subject to the following conditions:
+ *
+ *     The above copyright notice and this permission notice shall be included in
+ *     all copies or substantial portions of the Software.
+ *
+ *     THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ *     EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ *     OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ *     IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+ *     CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ *     TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE
+ *     OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package org.elasticsearch.discovery.srv;
+
+import org.elasticsearch.common.inject.Inject;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.discovery.Discovery;
+import org.elasticsearch.discovery.zen.ZenDiscoveryModule;
+
+public class SrvDiscoveryModule extends ZenDiscoveryModule {
```

Comment:
don't you need to return this module in the Plugin onModules or something like that?